java_tools/configuration_definition/src/com/rusefi/ConfigDefinition.java

```diff
@@ -325,6 +342,85 @@ public static void readPrependValues(VariableRegistry registry, String prependFi
         }
     }
 
+    public static void processYaml(VariableRegistry registry, File yamlInputFile, ReaderState state) throws IOException {
+        Yaml yaml = new Yaml();
+        Map<Integer, String> listOutputs = new HashMap<>();
+        Map<Integer, String> listAnalogInputs = new HashMap<>();
+        Map<Integer, String> listEventInputs = new HashMap<>();
+        Map<Integer, String> listSwitchInputs = new HashMap<>();
+        Map<String, Object> yamlData = yaml.load(new FileReader(yamlInputFile));
+        List<Map<String, Object>> data = (List<Map<String, Object>>) yamlData.get("pins");
```

Comment:
Idea says: Unchecked cast: 'java.lang.Object' to 'java.util.List<java.util.Map<java.lang.String,java.lang.Object>>'

What does this mean, and how do I resolve it?

Reply:
Second answer is more useful: https://stackoverflow.com/questions/262367/type-safety-unchecked-cast

> Use @SuppressWarnings("unchecked") and hold your nose. Oh, and campaign for reified generics in Java :)

Just put `@SuppressWarnings("unchecked")` on the line before the function definition (345-ish).